### ISSUE TITLE
F-Droid

### ISSUE THREAD

**Issue by Poussinou** [thibautperrin/QuoteReader#2]
Penses tu ajouter ton application sur F-Droid, un magasin d'applications sous licences libres? Si tu connais déjà je t'y encourages, sinon je peux te donner des liens vers de la doc;) il est probable que beaucoup de gens qui utilisent F-Droid et parlent français aiment ton application

**Comment by thibautperrin** [comment]
Oui, on m'en a parlé. D'un autre côté, les gens sur F-droid sont aussi sur
le playstore donc tout le monde a accès à l'app. Après c'est vrai que ça me
donnerait de la visibilité vers un public un peu plus geek, à voir ;-)

**Comment by Poussinou** [comment]
Un certain nombre de gens, les amateurs purs et durs de l'open source et du libre dont je fais presque parti (mais j'ai gardé le play store parce que quand même faut pas déconner ^^) n'utilisent que F-Droid, crois moi. Et beaucoup de gens qui ont F-Droid privilégieront ce magasin plutôt que le play store. Et puis ça laissera l'application dans un magasin, parce qu'un jour elle sautera peut être du play store^^ à toi de voir :-)
